<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAP-138" failures="1" tests="2" name="LoginProcessTest" time="15.07" errors="0" timestamp="2024-09-08T12:39:22 IST">
  <testcase classname="com.testwebsite.scripts.LoginProcessScript" name="loginToAKPMS" time="6.755">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Login failed with error: Invalid Credentials">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Login failed with error: Invalid Credentials
at com.testwebsite.scripts.LoginProcessScript.loginToAKPMS(LoginProcessScript.java:64)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- loginToAKPMS -->
  <testcase classname="com.testwebsite.scripts.LoginProcessScript" name="loginToAKPMS" time="6.719"/>
</testsuite> <!-- LoginProcessTest -->
